python-backend/app/api/document_manager/resources/document_manager.py

```diff
@@ -83,3 +83,14 @@ def post(self):
             'document_manager_guids' : document_guid_list,
             'errors': errors,
         }
+
+    @api.doc(params={'document_guid': 'Required: Document guid. Returns the file associated to this guid.'})
+    @jwt.requires_roles(["mds-mine-create"])
+    def get(self, document_guid):
+
+        if document_guid == None:
+            return self.create_error_payload(401, 'Must provide a document guid.')
+
+        document_manager_doc = DocumentManager.find_by_document_manager_guid(document_guid)
```

Comment:
What happens if this is `None`? Will `send_file()` throw an exception while accessing `file_display_name`?

Reply:
I noticed that as well and added it while I was looking at the tests or how to test the byte stream being returned.